raw_macaroons.go

```diff
@@ -0,0 +1,171 @@
+package lndclient
+
+import (
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+)
+
+// RawMacaroons contains raw []byte macaroons
+// for later hex encoding and use with LND.
+type RawMacaroons struct {
+	AdminMac     []byte
+	InvoiceMac   []byte
+	ChainMac     []byte
+	SignerMac    []byte
+	WalletKitMac []byte
+	RouterMac    []byte
+	ReadOnlyMac  []byte
+}
```

Comment:
I think it'd be better if these struct fields didn't "stutter". The current names contain "macaroon" twice, and I think it's fine to omit this from the field names. We'd then end up with `RawMacaroons.Admin` ++, which I think is nicer